frontend/src/metabase/auth/containers/ResetPasswordApp/ResetPasswordApp.tsx

```diff
@@ -0,0 +1,19 @@
+import { connect } from "react-redux";
+import ResetPassword from "../../components/ResetPassword";
+import {
+  resetPassword,
+  validatePassword,
+  validatePasswordToken,
+} from "metabase/auth/actions";
```

Comment:
Relative imports should go below the absolute ones by our style guide AFAIR

Reply:
Thanks, fixed!